Code/server/GameServer.cpp

```diff
@@ -58,6 +58,22 @@ Console::Command<> TogglePvp("TogglePvp", "Toggle PvP on/off", [](Console::ArgSt
     spdlog::get("ConOut")->info("PvP has been {}.", bEnablePvp == true ? "enabled" : "disabled");
 });
 
+Console::Command<int64_t> SetDifficulty("SetDifficulty", "Set server difficulty (0 being Novice and 5 being Legendary; default is 4)", [](Console::ArgStack& aStack) {
```

Comment:
Currently connected clients need a message telling them to change the difficulty, otherwise the difficulty will be desynced.

Reply:
Resolved by Sturlen in merged pr below